posh-profile.psm1

```diff
@@ -30,6 +30,8 @@ Function Save-History($fileNameOrPath)
 Import-Module posh-docker
 Import-Module Jump.Location
 Import-Module PoShFuck # Slightly vulgar Typo Correcter
+Import-Module ("module_macaddress.psm1")
```

Comment:
This does not work (even if you are in the same working directory). You can use e.g. `(Join-Path $PSScriptRoot "module_macaddress.psm1")` instead

Reply:
That is true, in my own powershell profile I use the entire directory path, something I forgot to include here. But using ```join-path``` is indeed the way to go here. Myself I use something like this: 
```
$Modules            = ("$($PSProfileDirectory)\Modules")
Import-Module ("$($Modules)\module_networking.psm1")    # Networking Tools
```